src/Form/DatasetSubmissionType.php

```diff
@@ -282,6 +282,12 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     ])
                 ]
             ))
+            ->add('datasetFilePath', Type\TextType::class, array(
```

Comment:
I'm not sure how we are going to do this, I think the file manager might be here again.

Reply:
I'm assuming the global ingest would be a folder path. On submission form POST, we get the path, populate fileset and file objects and then the filer would process the files. Instead of getting it from the upload folder, it would get from the globus ingest